yarn.lock

```diff
@@ -2,42 +2,42 @@
 # yarn lockfile v1
```

Comment:
I'm not maintainer, but I'll share my thoughts. Hope, that's okay:

In an earlier PR I removed the yarn.lock file, so we do not have to keep both lock files up to date (yarn.lock, package-lock.json). I guess, it would be enough to just use one package manager. I personally do not care, which one to use, but in my view, it should just be 1.